sdks/python/setup.py

```diff
@@ -252,6 +242,7 @@ def get_portability_package_data():
         'objsize>=0.6.1,<0.7.0',
         'pymongo>=3.8.0,<4.0.0',
         'proto-plus>=1.7.1,<2',
+        'protobuf>=4.00,<=5',
```

Comment:
1) Don't use <= in upper bounds.  Once version 5 is released, the version 5.0.0 will always be used by Beam SDK. If 5.0.0 is buggy, protobuf won't be able to produce a patch release that old version of Beam will pick up. See: https://docs.google.com/document/d/1gxQF8mciRYgACNpCy1wlR7TBa8zN-Tl6PebW-U8QvBk/edit?resourcekey=0-XcHRyFh4KRPkA0GsdUmU3g#heading=h.lev3iz8uzeuh

```suggestion
        # use a tighter upper bound in protobuf dependency
        # to make sure the minor version at job submission
        # does not exceed the minor version at runtime.
        # To avoid depending on an old dependency, update the minor version on
        # every Beam release, see: https://github.com/apache/beam/issues/23119
        'protobuf>=4.0.0,<4.23.0',
```

2) we will have to update the linked issue to also update protobuf dependency for each release.

Reply:
That should be < instead of <=. Thanks for catching it.